app/models/rules/service_need.rb

```diff
@@ -0,0 +1,9 @@
+class Rules::ServiceNeed < Rule
+  def clients_that_fit(scope, requirement, opportunity)
+    if Client.column_names.include?(:service_need.to_s)
+      scope.where(service_need: requirement.positive)
+    else
+      raise RuleDatabaseStructureMissing.new("clients.enrolled_in_th missing. Cannot check clients against #{self.class}.")
```

Comment:
Ah, this probably should say:
```suggestion
      raise RuleDatabaseStructureMissing.new("clients.service_need missing. Cannot check clients against #{self.class}.")
```

Reply:
I think I missed that on the last rule

Reply:
I've got this updated in the current PR and will create a new change for the housing barrier rule

Reply:
Thanks!